apps/api/v2/src/modules/oauth/oauth-client.controller.ts

```diff
@@ -66,8 +70,12 @@ export class OAuthClientController {
   @Get("/:clientId")
   @HttpCode(HttpStatus.OK)
   @Roles([MembershipRole.ADMIN, MembershipRole.OWNER, MembershipRole.MEMBER])
+  @UseFilters(HttpExceptionFilter)
```

Comment:
Do we need to add this if we have HttpEceptionFilter applied globally in app.ts ?

app.useGlobalFilters(new HttpExceptionFilter());